src/auto-evo/simulation/PopulationSimulation.cs

```diff
@@ -157,7 +157,9 @@ private static List<Species> CopyInitialPopulationsToResults(SimulationConfigura
                 new CompoundFoodSource(patch, Glucose),
                 new CompoundFoodSource(patch, HydrogenSulfide),
                 new CompoundFoodSource(patch, Iron),
```

Comment:
This shouldn't do anything in the current state of the game, but I like that you kept it in. +1 for versitility!